projects/plugins/jetpack/class.jetpack-admin.php

```diff
@@ -133,14 +133,18 @@ function get_modules() {
 								? substr( $short_desc, 0, 140 ) . '...'
 								: $short_desc );
 
-				$module_array['module']            = $module;
-				$module_array['activated']         = ( $jetpack_active ? in_array( $module, $active_modules ) : false );
-				$module_array['deactivate_nonce']  = wp_create_nonce( 'jetpack_deactivate-' . $module );
-				$module_array['activate_nonce']    = wp_create_nonce( 'jetpack_activate-' . $module );
-				$module_array['available']         = self::is_module_available( $module_array );
-				$module_array['short_description'] = $short_desc_trunc;
-				$module_array['configure_url']     = Jetpack::module_configuration_url( $module );
-				$module_array['override']          = $overrides->get_module_override( $module );
+				$module_array['module'] = $module;
+
+				$is_available = self::is_module_available( $module_array );
+
+				$module_array['activated']          = ( $jetpack_active ? in_array( $module, $active_modules, true ) : false );
+				$module_array['deactivate_nonce']   = wp_create_nonce( 'jetpack_deactivate-' . $module );
+				$module_array['activate_nonce']     = wp_create_nonce( 'jetpack_activate-' . $module );
+				$module_array['available']          = $is_available;
+				$module_array['unavailable_reason'] = $is_available ? '' : self::get_module_unavailable_reason( $module_array );
```

Comment:
Should we default to `false` here instead of an empty string, or update `get_module_unavailable_reason` so it returns an empty string when the module is available, to have consistent output everywhere?

Reply:
Yes! This should default to `false` when the module is available. This is to indicate that either the module is available or an error occurred in `get_module_unavailable_reason`. An empty string indicates that we probably don't have an explicit reason for the module being unavailable.